src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -257,21 +266,21 @@ else if(purpose.equals("Adding users")){
         return;
       }
 
+      if(userNameArray == null || userNameArray.length == 0){
+        response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
```

Comment:
It might be useful to return a suitable error message here along with setting the status. Something like:
`response.getWriter().write("Conversation must have at least one member.");`.
Ideally, the client code would also catch these errors and display them nicely to the user, but it's probably fine not to do that now since we're not really covering all error cases in client code. I'd say the server should still make sure it's sending reasonable error messages.